packages/wordpress-kit/src/lib/tailwindcssPlugin/components/Separator.ts

```diff
@@ -0,0 +1,34 @@
+export const SeparatorComponent = () => {
+	return {
+		'.wp-block-separator': {
+			border: 'none',
+			borderBottom: '2px solid',
+			marginLeft: 'auto',
+			marginRight: 'auto',
+			borderColor: 'black !important',
+			'&.is-style-default': {
+				width: '100px',
+			},
+			'&.is-style-wide': {
+				maxWidth: '650px',
```

Comment:
Looks like the wide separator match the None width on WP, that in the starter is 720px